packages/core/core/src/utils/nuxt/_proxyUtils.ts

```diff
@@ -1,13 +1,35 @@
 import { Context as NuxtContext } from '@nuxt/types';
 import merge from 'lodash-es/merge';
 import { ApiClientMethod } from './../../types';
+import { AxiosInstance } from 'axios';
 
 interface CreateProxiedApiParams {
   givenApi: Record<string, ApiClientMethod>;
   client: any;
   tag: string;
 }
 
+const getUrl = (nxtContent: NuxtContext, endpoint: string) => {
+  const { base, req } = nxtContent;
+  const isServer = process.server;
+  const isClient = process.client;
+  const hasApiTag = /\/api\//gi.test(endpoint);
+  const apiEndpoint = hasApiTag ? endpoint : `api/${endpoint}`;
```

Comment:
Shouldn't `api` be the last part of whole url?

Reply:
@bloodf the placement is wrong. Should be `${endpoint}/api` I guess.

Reply:
No @lukasborawski , if I do this, it will break, because the endpoint will be like `shopify/get-products`

Reply:
@bloodf What, how? `${endpoint}/api` = `shopify.com/api/get-products` and `shopify.com` is my endpoint, or even `shopify.com/api/` or `api.shopify.com` or whatever, that way you'll keep the security rules as well.